README.md

```diff
@@ -149,7 +85,7 @@ def ray_taskflow_dag():
         futures = [square.remote(x) for x in data]
         results = ray.get(futures)
         mean = np.mean(results)
-        print(f"Mean of squared values: {mean}")
+        print(f"Mean of this population is {mean}")
```

Comment:
Probably using a Python logger would be more recommended:
https://www.astronomer.io/docs/learn/logging

Reply:
This is an example Ray code that executes on the Ray cluster. All Airflow logging is done through self.log.info()